### PR TITLE
Update the link under "privileged mask argument"

### PR DESCRIPTION
**Pull request by s011110101**
The [old link](https://www.tensorflow.org/guides/understanding_masking_and_padding) is changed to the new [one](https://www.tensorflow.org/guide/keras/understanding_masking_and_padding).

### CODE REVIEW

**Comment by divyashreepathihalli** [comment]
The link updates are incorrect. The existing links seem to work.

**Review by fchollet — COMMENTED** [review]
The link in the code is correct -- it just refers to `keras.io`

**Comment by fchollet** [comment]
You could change it to an absolute link starting with `https://keras.io/`

File: guides/making_new_layers_and_models_via_subclassing.py [hunk 1]
```diff
@@ -446,7 +446,7 @@ def call(self, inputs, training=False):
 
 To learn more about masking and how to write masking-enabled layers, please
 check out the guide
-["understanding padding and masking"](/guides/understanding_masking_and_padding/).
+["understanding padding and masking"](/guide/keras/understanding_masking_and_padding).
```
Comment (MarkDaoust) [inline guides/making_new_layers_and_models_via_subclassing.py:449]:
```suggestion
["understanding padding and masking"](https://keras.io/guides/understanding_masking_and_padding/).
```

**Comment by MarkDaoust** [comment]
Hi, we can't merge this unless you sign the CLA. 

If you got to this as a broken link on tensorflow.org (this guides directory is mirrored there), I'm working on fixing all of those.

**Comment by s011110101** [comment]
Hmm that's weird. Sorry for the confusion!
